lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -770,6 +881,20 @@ class BitmapCanvas extends EngineCanvas {
   void endOfPaint() {
     _canvasPool.endOfPaint();
     _elementCache?.commitFrame();
+    // Wrap all elements in translate3d (workaround for webkit paint order bug).
+    if (_contains3dTransform && browserEngine == BrowserEngine.webkit) {
+      for (html.Element element in rootElement.children) {
+        html.DivElement paintOrderElement = html.DivElement()
+          ..style.transform = 'translate3d(0,0,0)';
```

Comment:
What if, instead of adding a div for every child we applies it to a common parent? Wouldn't that have the same effect?

Reply:
It didn't. z order got reversed again.